### ISSUE TITLE
Replace datetime picker with native one on new event modal

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/calendar#4626]
### Steps to reproduce

1. Click into your calendar grid to create an event
2. Pick start/end time

### Expected behavior

Native datetime picker

### Actual behaviour

![Bildschirmfoto vom 2022-10-10 09-34-46](https://user-images.githubusercontent.com/1374172/194819096-b6bbe649-9770-4c63-9cdb-9c92fca450d6.png)

### Calendar app version

4.0.0

**Comment by corbin-auriti** [comment]
Is anyone else working on this? Otherwise I would like to along with a "natural language" title datetime grabber similar to Elementary OS Calendar and others https://github.com/elementary/calendar/commit/8a2063007acd73cbecf8d5c6877aafe20ee8b214

**Comment by jancborchardt** [comment]
I checked with Firefox, also referencing here from https://github.com/nextcloud/calendar/issues/6385#issuecomment-2432512262

> On the Firefox side, the bug about a time picker is tracked as https://bugzilla.mozilla.org/show_bug.cgi?id=1811581 and/or https://bugzilla.mozilla.org/show_bug.cgi?id=451832
>
> So I would say we indeed just accept the tradeoff that Firefox does not have a visual time picker. The keyboard-only input is accessible at least, and then hopefully they will soon activate the visual timepicker.
>
> And as written in the issue, they actually have a timepicker interface, but it unfortunately is hidden behind the `about:config` switch `dom.forms.datetime.timepicker`. If set to true, you see the picker, e.g. on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time
> ![Image](https://github.com/user-attachments/assets/ee5cf201-6216-4166-8502-ba6c35613d35)
